executor/update.go

```diff
@@ -132,32 +123,18 @@ func (e *UpdateExec) canNotUpdate(handle types.Datum) bool {
 // Next implements the Executor Next interface.
 func (e *UpdateExec) Next(ctx context.Context, req *chunk.Chunk) error {
 	req.Reset()
-	if !e.fetched {
-		err := e.fetchChunkRows(ctx)
+	if !e.drained {
+		rows, err := e.updateRows(ctx)
```

Comment:
It is better to rename `rows` to others. `rows` usually mean some rows not the number for rows.

Reply:
renamed to `numRows`.